src/otx/algo/detection/heads/distance_point_bbox_coder.py

```diff
@@ -0,0 +1,86 @@
+""""Distance Point BBox coder."""
```

Comment:
I think in v2 license-docstring order is used, isn't it?

Reply:
Thanks. I changed the order accordingly.